.github/workflows/docker-release.yml

```diff
@@ -42,13 +57,52 @@ jobs:
           exit 1
         fi
         echo "VERSION=$VERSION" >> $GITHUB_ENV
-        
+
+        if [ "${{ github.event.inputs.repository-name }}" != "" ]; then
+          REPONAME=${{ github.event.inputs.repository-name }}
+        else
+          REPONAME="ElementsProject"
+        fi
+        echo "REPONAME=$REPONAME" >> $GITHUB_ENV
+
+        if [ "${{ github.event.inputs.platforms-to-build }}" != "" ]; then
+          PLATFORMS=${{ github.event.inputs.platforms-to-build }}
+        else
+          PLATFORMS="linux/amd64,linux/arm64,linux/arm/v7"
+        fi
+        echo "PLATFORMS=$PLATFORMS" >> $GITHUB_ENV
+
+        if [ "${{ github.event.inputs.push-latest }}" != "" ]; then
+          PUSHLATEST=${{ github.event.inputs.push-latest }}
+        else
+          PUSHLATEST="true"
```

Comment:
This contradicts the default from above.

Reply:
Thanks, fixed.